.pre-commit-config.yaml

```diff
@@ -1,4 +1,12 @@
 repos:
+  - repo: local
+    hooks:
+      - id: stash-all-files
```

Comment:
Ah, so only staged files will have precommit hook checks ran on them?  This probably fits my workflow better as I never do a `git add --all`  👍

Reply:
Correct! 😄